### DISCUSSION TITLE
Anyone else having trouble with Alexa recently?

### DISCUSSION THREAD

**Discussion by pkkrusty** [arendst/Tasmota#22626] [Q&A]
Since about the 8th, my 2nd gen Echo dots are unable to turn on and off most of my Tasmota devices. A few work fine. There doesn't seem to be any correlation between firmware version and success that I can tell. So not really a Tasmota issue, but wanted to see if anyone else was having trouble.

**Comment by s-hadinger** [comment]
Logs please. It works fine with me.

**Reply by pkkrusty** [reply]
I restarted everything and it's back to normal. I think it was probably a router issue.
